### ISSUE TITLE
Palinopsia

### ISSUE THREAD

**Issue by carolina-f** [obophenotype/human-phenotype-ontology#10590]
**Preferred term label:**

**Synonyms**

**Definition (free text, please give PubMed ID)**
PMID: 34671311 Palinopsia is an abnormal, continued perception of an object even after it is no longer in the visual field (18, 19). This symptom is reported in 33% to 86% of patients with VSS. it can manifest as a steady afterimage or as “trailing,” i.e., positive afterimages persisting directly after a dynamic object.

PMID: 36788360 Palinopsia in VSS can be described as illusory, meaning that it is caused by the misinterpretation of a real object [16]. It is reported in around 60–80% of patients, either as afterimages, light streaking or trailing of moving object

PMID: 29140814 Palinopsia. At least one of the following: afterimages or trailing of moving objects. After images should be different from retinal afterimages, which occur only when staring at a high contrast image and are in complementary color

**Parent term (use hpo.jax.org/app)**

Abnormality of vision HP:0000504
**Diseases characterized by this term ? (e.g. Orphanet or OMIM number)**
ORPHA:420556 Visual snow syndrome

**Your nano-attribution (ORCID)**
0009-0005-6714-5727

**Comment by pnrobinson** [comment]
@carolina-f 
Added as HP:0020333
